tests/README.rst

```diff
@@ -80,6 +80,125 @@ To do that, run `podman pull quay.io/rhinstaller/anaconda-ci:master` or build
+These files completely describe what steps are required to run the tests, what
+are the triggers and so on. We use GitHub runners here so we don't have to care
+too much about what is executed there. GitHub runners are designed to be isolated
+and secure. On master we are building the container only if the container files
+have changed, otherwise we are pulling the container from our repository. For more
+information see below.
+
+Pull request for RHEL:
+______________________
+
+Again configuration is stored in `.github/workflows/validate-rhel-8.yml`_ the
+workflow. However, here we also have (fully automatically deployed) runners in
+our Upshift instance. GitHub configuration is using our self-hosted runners.
+These runners are ``anaconda-ci:rhel8`` containers with all the dependencies
+in place so the yml configuration will just execute tests. 
+You can start runners locally by running the container and
+providing GitHub token. That is pretty valuable in case of workflow testing.
+See `github-action_run_once <https://github.com/rhinstaller/anaconda/blob/rhel-8/dockerfile/anaconda-ci/github-action-run-once>`_
+for more details.
+
+To protect our self-hosted runners, tests only run automatically for
+`rhinstaller organization members <https://github.com/orgs/rhinstaller/people>`_.
+For external contributors, an organization member needs to approve the test run
+by sending a comment starting with ``/tests``.
+
+
+What role do the containers have
+--------------------------------
+
+All active branches run tests in containers. Containers have all the
+dependencies installed and the environment prepared to run tests or connect our
+GitHub runners (used by RHEL only).
+
+Automatic container build
+_________________________
+
+Containers are updated daily by `.github/workflows/container-autoupdate.yml`_
+GitHub action from Anaconda ``master`` repository. Before pushing a new
+container, tests are executed on this container to avoid regressions.
+
+Manual container build
+______________________
+
+There are two possible ways how to build containers manually.
+
+GitHub: Just go to the
+`actions tab <https://github.com/rhinstaller/anaconda/actions?query=workflow%3A%22Refresh+container+images%22>`_
+in the Anaconda repository to the “Build, test and push container“ and press the
+``Run workflow`` button on a button on a particular branch. Usually ``master``,
+but for testing a change to the container you can push your branch to the
+origin repo and run it from there.
+
+Security precautions for testing RHEL
+-------------------------------------
+
+Getting into our host/internal network
+______________________________________
+
+One of the main precautions is that each container test run has
+a limited time and is destroyed after timeout/end of test. That should narrow
+what attackers could do or how they can create a backdoor. See the image for
+more info:
+
+.. image:: docs/images/tests/GH-self-hosted-runners.png
```

Comment:
Wrong link target, see your rendered result. It wants to show .../tests/docs/images/tests/GH-self-hosted-runners.png . I figure you need to do ../docs/ here for a proper relative path.

Reply:
It was test seems it failed and I forgot to check it. Sorry